lib/util/fs/index.ts

```diff
@@ -162,3 +163,34 @@ export function localPathIsFile(pathName: string): Promise<boolean> {
     .then((s) => s.isFile())
     .catch(() => false);
 }
+
+/**
+ * Find a file or directory by walking up parent directories within localDir
+ */
+
+export async function findUpLocal(
+  fileName: string | string[],
+  cwd: string
+): Promise<string | null> {
+  const { localDir } = GlobalConfig.get();
+  const absoluteCwd = upath.join(localDir, cwd);
+  const normalizedAbsoluteCwd = upath.normalizeSafe(absoluteCwd);
+  const res = await findUp(fileName, {
+    cwd: normalizedAbsoluteCwd,
+    type: 'file',
+  });
+  // Return null if nothing found
+  if (res === undefined) {
```

Comment:
```suggestion
  if (!!res || !!localDir) {
```

`localDir` should never null or undefined.